libs/design-system/src/panda/semanticSizes.tokens.ts

```diff
@@ -0,0 +1,18 @@
+import { defineSemanticTokens } from '@pandacss/dev';
+import { INPUT_HEIGHT_PX } from '../config/inputs.styles';
+
+/**
+ * Represents a dimension of an element.
+ *
+ * Used for properties like width and height.
+ */
+export const SEMANTIC_SIZES_TOKENS = defineSemanticTokens.sizes({
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** Excited for this! This made it easier to work with some hardcoded values that otherwise didn't work within Panda. We can change these as the design system matures, but I suggest we continue adding here as we encounter useful, hard-coded `px / rem` values‏